src/main/java/dk/cngroup/lentils/exception/NextCypherNotFoundException.java

```diff
@@ -0,0 +1,7 @@
+package dk.cngroup.lentils.exception;
+
+public class NextCypherNotFoundException extends RuntimeException {
+    public NextCypherNotFoundException() {
+        super("Status of the last cypher has been changed");
```

Comment:
Toto je spatna zprava. Ta aktualni ted rika, co predchazelo teto vyjimce a nepopisuje vyjimku samotnou. Tady musi byt jen neco jako "Next cypher not found.".